### ISSUE TITLE
Build out gateways page

### ISSUE THREAD

**Issue by pippinsplugins** [restrictcontentpro/rcp-site-child#35]
I was perusing the Paid Memberships Pro site and noticed they have a really nice Gateways page that describes each gateway in details:

Top level page: http://www.paidmembershipspro.com/gateway/

Detail page for Stripe: http://www.paidmembershipspro.com/gateway/stripe/

Detail page for PayPal: http://www.paidmembershipspro.com/gateway/paypal/

@sumobi I'd love to build out something like this. Thoughts?

**Comment by sumobi** [comment]
@pippinsplugins I like it

**Comment by pippinsplugins** [comment]
@sumobi Want to work with me on it in early 2016? Perhaps January?

**Comment by sumobi** [comment]
Absolutely

**Comment by sumobi** [comment]
Made a start to this.

Here's the main payment gateways page so far. I've made a new page template that lists all the child pages which we can re-use for other sections of the site if need be.

![screen shot 2015-12-28 at 4 31 03 pm](https://cloud.githubusercontent.com/assets/52581/12013655/810f80f2-ad80-11e5-8e51-088e58367700.png)

___

Here's the single payment gateway page so far. Currently working on styling the navigation on the right.

![screen shot 2015-12-28 at 4 31 16 pm](https://cloud.githubusercontent.com/assets/52581/12013654/810ea24a-ad80-11e5-95a3-fd3e134e2a93.png)

**Comment by sumobi** [comment]
Oh yeah and the images are SVGs :D

**Comment by sumobi** [comment]
Better version of the subpages menu

![screen shot 2015-12-28 at 8 05 19 pm](https://cloud.githubusercontent.com/assets/52581/12015235/6a93b208-ad9e-11e5-94b4-f9eb70fd9aeb.png)

**Comment by sumobi** [comment]
@pippinsplugins Can you see anything else we need for this?

**Comment by pippinsplugins** [comment]
Are the images controlled via featured image?

**Comment by sumobi** [comment]
@pippinsplugins sure are

**Comment by pippinsplugins** [comment]
Love it so much. Could you deploy to live and then create images (if not already done) for each of the gateways? I'll write up the pages.

**Comment by sumobi** [comment]
Absolutely. I've done the 3 images shown above. The PayPal image and page can cover Website payments pro, standard and express. We're just missing an image for 2Checkout by the looks of things, correct?

> Accept credit cards with Stripe, Braintree, 2Checkout, or PayPal Website Payments Pro. Restrict Content Pro also supports PayPal Standard and Express, as well as multiple payment options at the same time.

**Comment by sumobi** [comment]
We should link the logos on the homepage through to their respective pages once the payment gateways section is done.

**Comment by sumobi** [comment]
@pippinsplugins Go nuts http://restrictcontentpro.com/payment-gateways/

If you need to create any more, just create a new page and make it a child of the main payment gateways page. If you need anymore featured images, let me know.

**Comment by pippinsplugins** [comment]
Woot! I'll get them written up later this week!

**Comment by pippinsplugins** [comment]
First draft of Stripe written: http://restrictcontentpro.com/payment-gateways/stripe/

**Comment by sumobi** [comment]
@pippinsplugins looks great! except for the responsiveness, but that's a me thing. Will get it fixed.

**Comment by pippinsplugins** [comment]
2Checkout done: http://restrictcontentpro.com/payment-gateways/2checkout/

**Comment by pippinsplugins** [comment]
All gateways written up.

**Comment by sumobi** [comment]
@pippinsplugins you can close this after you've added your "adding a new gateway" page

**Comment by pippinsplugins** [comment]
Now need an image @sumobi, then let's close!
